google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -555,6 +555,40 @@ def delete force: nil
         #     end
         #   end
         #
+        # @example With time partitioning and clustering.
+        #   require "google/cloud/bigquery"
+        #
+        #   bigquery = Google::Cloud::Bigquery.new
+        #   dataset = bigquery.dataset "my_dataset"
+        #
+        #   table = dataset.create_table "my_table" do |table|
```

Comment:
I find it confusing to have a block variable shadow an outer variable of the same name. Can you rename one of the variables? (Same for the other example below.)

Reply:
This was done in 11 places. I'll change them all to `table = dataset.create_table "my_table" do |t|`.

Reply:
Updated.